src/lib/utils.jl

```diff
@@ -25,6 +25,23 @@ Obviously, you run the risk of incorrect gradients if you use this incorrectly.
 ignore(f) = f()
 @adjoint ignore(f) = ignore(f), _ -> nothing
 
+"""
+    @ignore (...)
+
+Tell Zygote to ignore an expression. Equivalent to `ignore() do (...) end`.
+Example:
+
+```julia-repl	
+julia> f(x) = (y = Zygote.@ignore x; x * y); f'(1)
```

Comment:
Would be great to have a slightly more realistic example here – and it's allowed to use multiple lines.

Reply:
I think this gets it accros but if you suggest anything I'll add it. I did fix the newline.